.github/workflows/update-workflows.yaml

```diff
@@ -0,0 +1,130 @@
+# This workflow gets copied to .github/workflows in the target repository. From there, it updates itself and other workflows from
+# templates in the template repository. The update runs once a day or on demand, and it creates a PR if there are changes to be
+# applied.
+
+# The target repo must have a WORKFLOW_PAT secret containing a GitHub personal access token with "workflow" permission.
+
+# When you install this in a repository, please ensure there's a .github/CODEOWNERS file to give @IronCoreLabs/ops reponsibility
+# for files under the .github directory.
+
+name: Update Workflows
+
+# "on" must be quoted, otherwise YAML will decide its tag is !!bool instead of !!str and change it to its canonical "true" form.
+'on':
+  push:
+    paths:
+    - .github/**
+    branches:
+    - '!main'
+  schedule:
+    - cron: 34 7 * * 1-5
+  workflow_dispatch: null
+
+env:
+  TEMPLATE_REPO: IronCoreLabs/depot
+  TEMPLATE_DIR: github-actions
+  # Space-separated list of repo workflows to keep updated.
+  WORKFLOWS: update-workflows.yaml
```

Comment:
You didn't manually apply the patch to this line. That's not a big deal; it'll work itself out after a run or two.

And I think I need to write a script to do this as part of the bootstrap process...